packages/styleguide/src/components/Page/Page.js

```diff
@@ -9,12 +9,36 @@ import { H1, H2, H3, H4, H5, P, Link } from './../../components/Typography';
 
 import { MDXProvider } from '@mdx-js/tag';
 
+/**
+ * Create custom header id which can be used as hash in link
+ * skips any react element in string and uses just strings
+ * id contains only lowercase characters ("\w") numbers ("\d") and dashes ("-") for separation
+ * Example: Hello 1 - anchor element, all you need
+ * Turns into: hello-1-anchor-element-all-you-need
+ */
+const createHeaderId = props => {
+  const children = props.children;
+  const text = Array.isArray(children)
```

Comment:
Inak tu by sa mozno hodilo pouzit `React.Children.toArray`, ono ti to vzdy spravi z children array.

Reply:
je to tam